modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -148,4 +199,71 @@ private String getFullClassName(Element classElement) {
         }
         return className.toString() + classElement.getSimpleName();
     }
+
+    private String getMethodIdentifierString(Element methodElement) {
+        List<? extends TypeMirror> methodParams =
+            ((ExecutableType) methodElement.asType()).getParameterTypes();
+        String fullClassPattern =
+            getClassHeader(methodElement.getEnclosingElement());
+        String fullClassName =
+            fullClassPattern.substring(0, fullClassPattern.indexOf('('));
+        String methodSignature = "";
+        if (methodElement.getKind().toString().equals("CONSTRUCTOR")) {
+            int index = fullClassName.lastIndexOf("\\$");
+            if (index != -1) {
+                methodSignature +=
+                    "L" + fullClassName.substring(0, index) + ";";
+            }
+        }
+        methodSignature += getMethodSignature(methodParams);
+        return fullClassName + "," +
+            methodElement.getSimpleName().toString() + "," +
+            methodSignature.replace("\\$", "$");
+    }
+
+    private String getMethodSignature(List<? extends TypeMirror> methodParams) {
+        if (methodParams.isEmpty()) {
+            return "";
+        }
+        String methodSignature = "";
+        for (TypeMirror param : methodParams) {
+            if (param.getKind().name().equals("TYPEVAR")) {
+                methodSignature += "Ljava/lang/Object;";
+                continue;
+            }
+            String paramString = param.toString();
```

Comment:
I'm concerned about relying on the output of `toString()`, is there another way to get the info we need?